### ISSUE TITLE
EEPROM library (energia 18b6)

### ISSUE THREAD

**Issue by reaper7** [energia/tivac-core#8]
I try to compile sketch with EEPROM lib:

```
#include <EEPROM.h>

void setup() {

}

void loop() {
  EEPROM.write(0, 1);
}
```

and I get error message:

```
C:\tmp\arduino_modified_sketch_695733\sketch_aug02b.ino: In function 'void loop()':

sketch_aug02b:8: error: 'EEPROM' was not declared in this scope

   EEPROM.write(0, 1);

   ^

exit status 1
'EEPROM' was not declared in this scope
```

**Comment by JLaferri** [comment]
I'm having the same issue. Did you find a solution?

**Comment by robertinant** [comment]
Thanks for the report. There is an issue with the include path in the file platform.txt.
Below is the work-around:

The paths below are:
1. On GNU/Linuxes: ~/.arduino15/packages/
2. On Windows: %APPDATA%\Arduino15\packages\
3. On MacOSXes: ~/Library/Arduino15/packages/

Then edit the file platform.txt located in the directory above plus energia/hardware/tivac/1.0.2/. This directory contains a file called platform.txt.

Look for:
`compiler.driverlib.c.flags="-I{build.system.path}/driverlib" "-I{build.system.path}/in    c" "-I{build.system.path}"`
And replace with:
`compiler.driverlib.c.flags="-I{build.system.path}"`

**Comment by robertinant** [comment]
FIxed in 1.0.3
